code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/contentprovider/AepUIContentProvider.kt

```diff
@@ -21,9 +21,9 @@ import kotlinx.coroutines.flow.Flow
 interface AepUIContentProvider {
     /**
      * Retrieves the content for the UI.
-     * @return The content for the UI as a flow of [AepTemplate]s.
+     * @return The content for the UI as a flow of [AepUI]s.
      */
-    suspend fun getContent(): Flow<List<AepUITemplate>>
+    suspend fun getContent(): Flow<List<AepUI<*, *>>>
```

Comment:
Why did we change this interface? It needs to be as is to be used by container, you can add an additional function in Messaging implementation if you want